### PR TITLE
Add JSON link in swagger description

### PR DESCRIPTION
**Pull request by LuciaAccenture**
## Pull Request Checklist

- [ ] **Title**
- [ ] **MOB US' Link**
- [ ] **Link PR to US**
- [ ] **Add Tests**
- [ ] **Add ENV Variables and Validations**
- [ ] **Check Story Points**
- [ ] **Documentation Updates**
- [ ] **Swagger Updates**

## Testing Guide

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -19,7 +19,9 @@ async function main() {
 
   const swaggerConfig = new DocumentBuilder()
     .setTitle('Esmorga API')
-    .setDescription('Swagger for Esmorga API.')
+    .setDescription(
+      '[Swagger in JSON format]( https://qa.esmorga.canarte.org/swagger-json)  \n\nSwagger for Esmorga API.',
```
Comment (JM-Abrodos) [inline src/main.ts:23]: Each enviroment has it own swagger, you can not add the QA swagger for all envs

**Review by JM-Abrodos — DISMISSED** [review]

**Review by alessandradosil — DISMISSED** [review]
It looks good to me, great job! Thanks! 👍

File: src/main.ts [hunk 2]
```diff
@@ -17,11 +17,23 @@ async function main() {
     }),
   );
 
-  const swaggerConfig = new DocumentBuilder()
+  let description = 'Swagger for Esmorga API.';
```
Comment (JM-Abrodos) [inline src/main.ts:20]:
```
  const description = process.env.DNS_NAME
    ? `[Swagger in JSON format](${process.env.DNS_NAME}/swagger-json)\n\n Swagger for Esmorga API.`
    : 'Swagger for Esmorga API.';

  const swaggerConfigBuilder = new DocumentBuilder()
    .setTitle('Esmorga API')
    .setVersion('1.0')
    .addServer(process.env.DNS_NAME)
    .setDescription(description);
```

**Comment by JM-Abrodos** [comment]
Pls add `DNS_NAME` value to PR description

File: .github/workflows/WhenPR.yml [hunk 3]
```diff
@@ -18,6 +18,7 @@ jobs:
       MONGODB_URI: "mongodb://localhost:27017/"
       REDIS_HOST: "redis-10358.c6.eu-west-1-1.ec2.redns.redis-cloud.com"
       REDIS_PORT: 10358
+      SWAGGER_JSON_SCHEMA_URL: "https://qa.esmorga.canarte.org/swagger-json"
```
Comment (JM-Abrodos) [inline .github/workflows/WhenPR.yml:21]: With the last change, this variable is not needed. You are using `DNS_NAME`

**Pull request closed without merging**